packages/frontend/src/components/UserSettings/ProfilePanel/index.tsx

```diff
@@ -202,26 +186,26 @@ const ProfilePanel: FC = () => {
                         .
                     </Text>
                 ) : null}
-            </FormGroup>
-
-            <div style={{ flex: 1 }} />
 
-            <Button
-                style={{ alignSelf: 'flex-end', marginTop: 20 }}
-                intent={Intent.PRIMARY}
-                text="Update"
-                onClick={handleUpdate}
-                loading={isLoading}
-                data-cy="update-profile-settings"
-            />
-            <VerifyEmailModal
-                opened={showVerifyEmailModal}
-                onClose={() => {
-                    setShowVerifyEmailModal(false);
-                }}
-                isLoading={statusLoading || emailLoading}
-            />
-        </div>
+                <Button
+                    type="submit"
+                    ml="auto"
+                    display="flex"
```

Comment:
is flex necessary here?

Reply:
the button's default `display` is `inline-block` so something like `flex` or `block` is necessary to move it to the right with `ml="auto"`. changed it to `display="block"`!

Reply:
cool!